src/core/icons/folderIcons.ts

```diff
@@ -1,1027 +1,1031 @@
-import type { FolderTheme } from '../models/icons/folders/folderTheme';
-import { IconPack } from '../models/icons/iconPack';
+import type { FolderTheme } from "../models/icons/folders/folderTheme";
```

Comment:
You formatted the whole document with another formatter. This makes the review more confusing and the git blame will be all on this commit. Would you mind reverting this formatting so that only your changes will be part of this commit?

Reply:
Sorry! Formatter's fault, and my fault 😅
I have formatted the document correctly and uploaded the fixed icons.

Reply:
Did you push your changes? I can't see some new changes/commits yet in here:
https://github.com/material-extensions/vscode-material-icon-theme/pull/2690/files

Reply:
Now? Sorry, it's my first PR ever.